src/components/Modal/OpenModal.tsx

```diff
@@ -21,9 +30,17 @@ import { useModal } from "src/components/Modal/useModal";
  * And `OpenModal` will do a boilerplate `openModal` call, so that the content
  * shows up in the DOM as expected.
  */
-export function OpenModal(props: { children: JSX.Element; size?: ModalProps["size"] }): JSX.Element {
+export function OpenModal(props: OpenModalProps): JSX.Element {
   const { openModal } = useModal();
-  const { size, children } = props;
-  useEffect(() => openModal({ size, content: children }), [openModal, size, children]);
-  return <div>dummy content</div>;
+  const { size, children, keepOpen } = props;
+  useEffect(() => {
+    if (!keepOpen) {
+      openModal({ size, content: children });
+    }
+  }, [keepOpen, openModal, size, children]);
+  if (keepOpen) {
+    return <Modal size={size} content={children} />;
+  } else {
+    return <div>dummy content</div>;
```

Comment:
was this just necessary to appease a consistent return type?

Reply:
Hm, no? This is just what it was before, i.e. the previous "keepOpen=false" behavior was to return some dummy content that is the pretend/top-level page content that then `openModal` renders on top of.